Makefile

```diff
@@ -158,7 +159,8 @@ ifneq ($(origin DOCKER_CMD),undefined)
 	$(DOCKER_CMD) make devtest TESTS="$(TESTS)"
 else
 #        the 'dropdb' command may fail, hence the prefix minus-sign
-	-dropdb lsmb_test
+	-perl -Ilib bin/ledgersmb-admin destroy \
```

Comment:
But dropdb works everywhere except on coverage testing CI. (And it never worked not was even required on CI because the database doesn't exist yet). I like the simplicity of dropdb.

Reply:
Fixed, the API tests will use another sample database